htdocs/modules/system/themes/modern/tests/test-theme.sh

```
@@ -0,0 +1,211 @@
+fi
+
+# Test 4: Template Syntax
+echo "→ Testing Smarty template syntax..."
+TEMPLATE_OK=true
+for tpl in "$THEME_DIR"/xotpl/*.tpl; do
+    # Check for balanced Smarty tags
+    OPEN=$(grep -o '<{' "$tpl" | wc -l)
+    CLOSE=$(grep -o '}>' "$tpl" | wc -l)
+
+    if [ "$OPEN" -ne "$CLOSE" ]; then
+        echo "  ⚠ WARNING - Unbalanced tags in $(basename $tpl)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)